### PR TITLE
Dev2

### PR DESCRIPTION
**Pull request by ali-tunahan**

### CODE REVIEW

**Comment by atakanserifoglu** [comment]
!cond_accept

**Comment by tech-extremists** [comment]
!cond_accept

**Comment by tech-extremists** [comment]
!cond_accept

File: .github/workflows/pr_agent.yml [hunk 1]
```diff
@@ -18,3 +18,5 @@ jobs:
         env:
           OPENAI_KEY: ${{ secrets.OPENAI_KEY }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          github_action_config.auto_review: "true" # enable\disable auto review
```
Comment (tech-extremists) [inline .github/workflows/pr_agent.yml:21]:
```suggestion
             github_action_config.auto_review: "true" # enable\disable auto review
```

**Comment by tech-extremists** [comment]
!cond_accept

**Pull request closed without merging**